document/core/syntax/modules.rst

```diff
@@ -261,7 +261,7 @@ The |MELEM| component of a module defines a vector of element segments. Each act
 .. math::
    \begin{array}{llll}
    \production{element segment} & \elem &::=&
-     \{ \ETABLE~\tableidx, \EOFFSET~\expr, \EINIT~\vec(\elemexpr) \} \\&&|&
+     \{ \ETABLE~\tableidx, \EOFFSET~\expr, \ETYPE~\elemtype, \EINIT~\vec(\elemexpr) \} \\&&|&
```

Comment:
Have you adjusted all the use sites of this record def?

Reply:
I checked it, yes. I don't know for sure though if I found all use sites.

Reply:
Shouldn't this at least require a change to the corresponding binary format decoding rule as well?

Reply:
This was already done in https://github.com/WebAssembly/bulk-memory-operations/pull/108.

Reply:
Ah sorry, I lost track of the changes. So I guess we should already have changed the AST in that PR, but I missed that while reviewing.

Anyway, LGTM then.